.changeset/quiet-camels-design.md

```diff
@@ -0,0 +1,5 @@
+---
+"@callstack/repack": major
+---
+
+Module Resolution Errors for ScriptManager Imports in FederationRuntimePlugin
```

Comment:
feels a bit too vague - I think something like "Fix bad import in MF runtime plugins" would be more descriptive, WDYT?

Reply:
https://github.com/callstack/repack/pull/833/commits/67a8ed55f85f8863a98ba4f67352134d4b1c45bf